src/vos/vos_tls.h

```diff
@@ -128,7 +128,12 @@ vos_dth_set(struct dtx_handle *dth)
 static inline struct dtx_handle *
 vos_dth_get(void)
 {
-	return vos_tls_get()->vtl_dth;
+	struct vos_tls	*tls = vos_tls_get();
```

Comment:
why is this needed?

Reply:
If without that, we will hit NULL pointer trouble when run vos evtree test.